src/BenchmarksApps/Grpc/GoServer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:alpine AS build-env
```

Comment:
I added the go benchmark code here because it doesn't make sense to add them to our usual grpc-dotnet repo.